docs/geedocs/5.2.5/answer/7160006.html

```diff
@@ -68,6 +68,15 @@ <h5>
             <th class="narrow">Description</th>
             <th>Resolution</th>
           </tr>
+          <tr>
+            <td>837</td>
```

Comment:
Since this is an enhancement and not a bug fix per say, let's document it under **New Features** section

Reply:
Sounds good, I’ll move it over and fix the wording this evening.